packages/types/src/atoms/graphql-types.ts

```diff
@@ -1505,12 +1510,13 @@ export type EhrConditionsArgs = {
   clinicalStatus?: Maybe<Array<Maybe<Scalars['String']>>>;
   count?: Maybe<Scalars['Int']>;
   cursorToken?: Maybe<Scalars['String']>;
-  isRelevantToSession?: Maybe<Scalars['Boolean']>;
```

Comment:
Did you mean to remove this?

Reply:
Yeah, previously getting pointInTime and filtering from isRelevantToSession. We deemed it that we need to differentiate between when we want too apply pointInTime and when we want to filter conditions (via the scope type)